decidim-core/lib/tasks/upgrade/decidim_fix_nickname_uniqueness.rake

```diff
@@ -2,9 +2,31 @@
 
 namespace :decidim do
   namespace :upgrade do
+    desc "Modifies nickname of the user to lower case"
```

Comment:
As this is a manual action for implementers, we would need to add an entry in the Releases Notes